### DISCUSSION TITLE
How do I explicitly define a password for RCON?

### DISCUSSION THREAD

**Discussion by manuelmolinaf** [thijsvanloef/palworld-server-docker#171] [Q&A]
Before it didn't require a password  when accessing rcon with the docker commando in the readme , but now its saying authentication failed for some reason. I did see that the rcon client got replaced. But anyways, i'd still like to set a password for rcon, is this possible right now?

**Comment by thijsvanloef** [comment]
@manuelmolinaf Are you using version 0.17.0 of the docker image? I'm pretty sure that if you are on the latest version, you would not have to enter a password for RCON

**Reply by manuelmolinaf** [reply]
Hey, just updated and you're right, it no longer says authentication failed, thanks!
